src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php

```diff
@@ -114,6 +116,10 @@ public function configureViews(ViewCollection $viewCollection): void
 
             if ($this->securityChecker->hasPermission(static::CONTACT_SECURITY_CONTEXT, PermissionTypes::VIEW)) {
                 $contactListToolbarActions[] = new ToolbarAction('sulu_admin.export');
+                $contactDocumentsItemActions[] = new ListItemAction(
+                    'link',
+                    ['icon' => 'fa-cloud-download', 'link_property' => 'url']
```

Comment:
There is a `su-download` icon :slightly_smiling_face:

Reply:
will use that one